zrp/modeling/pipeline_builder.py

```diff
@@ -190,9 +275,13 @@ class ZRP_DataSampling(BaseZRP):
         Indicates the source of zrp_modeling data to use. There are three optional sources 'block_group', 'census_tract', and 'zip_code'. By default 'census_tract' is inferred.
     population_weights_dict: dict
         Prevalence of target classes within the USA population as provided by the end-user. Sum of the values provided in the dictionary must be equal to one. Example: {'class1': 0.7, 'class2': 0.3}
+    test_size: float (default=0.2)
+        The fraction of samples to use as the test holdout
+    valid_size: float (default=0.2)
+        The fraction of samples to use as the test holdout
```

Comment:
`valid_size` update docstring

Reply:
@kasey-zest by default though there is no validation set used... hence why the default is None

Reply:
I switched it to default valid_size=0.0